sdk/storage/storage-blob/test/pageblobclient.spec.ts

```diff
@@ -131,7 +131,8 @@ describe("PageBlobClient", () => {
     assert.equal(page2.pageRange![0].end, 1023);
   });
 
-  it("getPageRangesDiff", async () => {
+  // This is the test failing
+  it.only("getPageRangesDiff", async () => {
```

Comment:
This is the test that is failing, specifically the check below at line 151
```ts
assert.equal(rangesDiff.clearRange![0].start, 512);
```
![image](https://user-images.githubusercontent.com/21109913/67031778-1b60d600-f0c7-11e9-848e-82adc6c8f556.png)

@jeremymeng, @HarshaNalluru  - Any thoughts?

Reply:
probably we got the result from the <PageRange> element.